packages/generator-adp/src/translations/generator-adp.i18n.json

```diff
@@ -40,28 +40,28 @@
         "telemetry": "An error occurred when sending telemetry data: {{- error}}. To see the error, view the logs.",
         "updatingApp": "An error occurred when creating a new adaptation project. To see the error, view the logs.",
         "cloudSystemsForInternalUsers": "You have selected a system that does not support On-Premise adaptation projects. Please select a supported system.",
         "notDeployableNotFlexEnabledSystemError": "The system that you have selected is not an ABAP On-Premise system. Adaptation projects are only supported on those systems. Please choose an ABAP On-Premise system which supports flexibility and DTA_FOLDER deployment.",
-        "notDeployableSystemError": "The system that you have selected is not an ABAP On-Premise system which supports DTA_FOLDER deployment. Adaptation projects are only supported on those systems. Please choose an ABAP On-Premise system which supports DTA_FOLDER deployment.",
+        "notDeployableSystemError": "The system that you have selected is not an ABAP On-Premise system which supports `DTA_FOLDER` deployment. Adaptation projects are only supported on those systems. Please choose an ABAP On-Premise system which supports `DTA_FOLDER` deployment.",
         "notFlexEnabledError": "The system that you have selected is not an ABAP On-Premise system which supports flexibility. Adaptation projects are only supported on those systems. Please choose an ABAP On-Premise system which supports flexibility. If you continue, you will only be able to create an extension project.",
-        "manifestCouldNotBeValidated": "The manifest.json file of the selected application could not be validated. Please select a different application.",
-        "appDoesNotSupportFlexibility": "The selected application does not support flexibility as it has (`flexEnabled=false`). SAPUI5 Adaptation Project only supports applications that support flexibility. Please select a different application.",
-        "appDoesNotSupportManifest": "The selected application is not supported by SAPUI5 Adaptation Project as it does not have a manifest.json file. Please select a different application.",
-        "extensibilityGenNotFound": "Extensibility Project generator plugin was not found in your dev space, and is required for this action. To proceed, please install the <SAPUI5 Layout Editor & Extensibility> extension.",
+        "manifestCouldNotBeValidated": "The `manifest.json` file of the selected application cannot be validated. Please select a different application.",
+        "appDoesNotSupportFlexibility": "The selected application does not support flexibility because it has (`flexEnabled=false`). SAPUI5 Adaptation Project only supports applications that support flexibility. Please select a different application.",
+        "appDoesNotSupportManifest": "The selected application is not supported by SAPUI5 Adaptation Project because it does not have a `manifest.json` file. Please select a different application.",
```

Comment:
Difference
`adp-tooling` uses:
“...not supported by SAPUI5 Adaptation Project as it does not have...”
`generator-adp` uses:
“...not supported by `SAPUI5 Adaptation Project` because it does not have...”
The only difference is the use of "as" vs "because" in the sentence.
The rest of the text is identical.